lookout/style/typos/cmdline_tools.py

```diff
@@ -147,6 +148,32 @@ def add_parser(name, help):
     train_from_scratch_parser.set_defaults(handler=train_from_scratch)
     add_config_arg(train_from_scratch_parser)
 
+    # Report for Typo Commits Dataset
+    typo_commits_report_parser = add_parser("typo-commits-report",
+                                            "Generate report for Typo Commits Dataset.")
+    typo_commits_report_parser.set_defaults(handler=generate_typos_report_entry)
+    add_config_arg(typo_commits_report_parser)
+    typo_commits_report_parser.add_argument(
+        "-i", "--dataset", required=True,
+        help="csv file with commits with typo. Must contain wrong_id, correct_id, file, line, "
```

Comment:
with typo**s**

Reply:
there is only one typo per commit in our dataset. Are you sure?

Reply:
Because of a data mining bug, which is fixed.